ocaml/idl/datamodel.ml

```diff
@@ -18,7 +18,7 @@ open Datamodel_types
 (* IMPORTANT: Please bump schema vsn if you change/add/remove a _field_.
               You do not have to bump vsn if you change/add/remove a message *)
 let schema_major_vsn = 5
-let schema_minor_vsn = 100
+let schema_minor_vsn = 101
```

Comment:
Looks like this whole patch contains fixups to the previous patches, so it should be folded into those.